components/Reference/index.tsx

```diff
@@ -82,7 +82,9 @@ const ReferenceTable = ({
     (opcodeOrAddress: string) => {
       // @ts-ignore: TODO: need to implement proper selection of doc according to selected fork (maybe similar to dynamic gas fee)
       // @ts-ignore: Hack for "difficulty" -> "prevrandao" replacement for "merge" HF
-      return opcodeOrAddress == '44' && selectedFork?.name === mergeHardforkName
+      return opcodeOrAddress == '44' &&
+        selectedFork?.name != null &&
+        postMergeHardforkNames.indexOf(selectedFork?.name) > -1
```

Comment:
nit: since the fork check is also done in `ethereumContext.tsx` it would be nice to just have a simple function with that logic

Reply:
makes sense 👍